$NetBSD: patch-aa,v 1.3 2007/10/01 23:39:21 dmcmahill Exp $

cp1250 seems to be a bit more widely supported.  Fixes unknown encoding problems
on netbsd-2

--- fo/biblio-iso690.xsl.orig	2007-06-28 23:19:07.000000000 +0000
+++ fo/biblio-iso690.xsl
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="windows-1250"?>
+<?xml version="1.0" encoding="cp1250"?>
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
                 xmlns:fo="http://www.w3.org/1999/XSL/Format"
                 version='1.0'>
